zephyr/Kconfig

```diff
@@ -77,4 +77,10 @@ config SOF_BOOT_TEST
 	  initialized. After that SOF will continue running and be usable as
 	  usual.
 
+config TELEMETRY
+	bool "enable telemetry"
+	default n
+	help
+	  Enable Telemetry
```

Comment:
This should probably be in a subdirectory too?

Reply:
Added Kconfig to src/debug